test/functional/pdo_sqlsrv/PDO100_InsertNulls.phpt

```diff
@@ -5,74 +5,82 @@ Test inserting nulls into nullable columns
 --ENV--
 PHPT_EXEC=true
 --SKIPIF--
-<?php require('skipif.inc'); ?>
+<?php require_once('skipif_mid-refactor.inc'); ?>
 --FILE--
 <?php
-include 'MsCommon.inc';
+require_once("MsCommon_mid-refactor.inc");
 
-function InsertNullsTest($bindType)
+function insertNullsTest($bindType)
 {
-    include 'MsSetup.inc';
-  
     $outvar = null;
-
     $failed = false;
-
-    Setup();
-
-    $conn = Connect();
-
-    DropTable($conn, $tableName);
-
-    CreateTable($conn, $tableName);
-
-    $stmt = $conn->query(<<<SQL
-SELECT [TABLE_NAME],[COLUMN_NAME],[IS_NULLABLE] FROM [INFORMATION_SCHEMA].[COLUMNS] WHERE [TABLE_NAME] = '$tableName'
-SQL
-);
-
-    if ($stmt === false)
-    {
+    $conn =connect();
+
+    $tableName = "pdo_test_table";
+    $dataType = array("c1_int" => "int",
+                      "c2_tinyint" => "tinyint",
```

Comment:
Please put a space character before `connect()`
Since this `$dataType` is an array, should probably use plurals or any name that implies it's not singular